Src/coffeescript/cql-execution/src/elm/quantity.coffee

```diff
@@ -10,6 +10,10 @@ module.exports.Quantity = class Quantity extends Expression
 
   exec: (ctx) ->
     @
+  
+  toString: () ->
+    "#{@value}#{@unit}"
+  
```

Comment:
Fixed in https://github.com/cqframework/clinical_quality_language/pull/35/commits/4732809275ed36d328d657bb7ce014cd7403d37e